driver-core/src/test/java/com/datastax/driver/core/policies/AbstractRetryPolicyIntegrationTest.java

```diff
@@ -113,18 +114,18 @@ protected void simulateError(int hostNumber, Result result) {
     protected void simulateError(int hostNumber, Result result, Config config) {
         PrimingRequestBuilder builder = PrimingRequest.queryBuilder()
                 .withQuery("mock query")
-                .withResult(result);
+                .withThen(then().withResult(result));
```

Comment:
Guessing test failures have to do with this, which is odd since it should be functionaly the same.  Maybe an issue with scassandra...diving..